e2e/fixtures/fdb_cluster_specs.go

```diff
@@ -178,7 +178,7 @@ func (factory *Factory) createPodTemplate(
 					ImagePullPolicy: factory.getImagePullPolicy(),
 					Resources:       fdbPodResources,
 					SecurityContext: &corev1.SecurityContext{
-						Privileged:               pointer.Bool(true),
+						//Privileged:               pointer.Bool(true),
```

Comment:
Same here, can we make this configurable instead of just commenting out? I believe we had some issues with chaos-mesh without this.